manta-trusted-setup/Cargo.toml

```diff
@@ -42,18 +45,25 @@ std = ["manta-util/std"]
 test = ["manta-crypto/test"]
 
 [dependencies]
+ark-bn254 = { version = "0.3.0", optional = true, default-features = false, features = ["curve", "scalar_field"] }
 ark-groth16 = { version = "0.3.0", default-features = false }
 ark-poly = { version = "0.3.0", default-features = false }
+ark-relations = { version = "0.3.0", default-features = false }
 ark-std = { version = "0.3.0", default-features = false }
 bincode = { version = "1.3.3", optional = true, default-features = false }
 blake2 = { version = "0.10.4", default-features = false }
-byteorder = { version = "1.4.3", default-features = false }
 derivative = { version = "2.2.0", default-features = false, features = ["use_core"] }
 manta-crypto = { path = "../manta-crypto", default-features = false, features = ["arkworks", "getrandom", "rand_chacha"] }
 manta-util = { path = "../manta-util", default-features = false }
+memmap = { version = "0.7.0", optional = true, default-features = false }
```

Comment:
We can also move this to `dev-dependencies` now since we are moving to slices rather than memmaps.